### ISSUE TITLE
Add SSIM as an option for loss + metrics

### ISSUE THREAD

**Issue by jennyfolkesson** [czbiohub/microDL#52]
https://www.tensorflow.org/api_docs/python/tf/image/ssim
https://stackoverflow.com/questions/48744945/keras-ms-ssim-as-loss-function

**Comment by jennyfolkesson** [comment]
@smguo has added dssim-loss to custom losses and ssim to metrics.
